samples/drivers/uart/native_tty/README.rst

```diff
@@ -0,0 +1,95 @@
+.. _sample-uart-native-tty:
+
+Native TTY UART
+###############
+
+Overview
+********
+
+This sample demonstrates the usage of Native TTY UART API. By using two UART to
```

Comment:
Maybe more readable as something like this?:
"This sample demonstrates the usage of Native TTY UART driver.
It uses two UART to USB bridge dongles that are wired together,
writing demo data to one dongle and reading it from the another."